src/Symfony/Component/Messenger/Worker.php

```diff
@@ -157,7 +159,7 @@ private function handleMessage(Envelope $envelope, ReceiverInterface $receiver,
                 $receiver->ack($envelope);
             } else {
                 if (null !== $this->logger) {
-                    $this->logger->critical('Rejecting {class} (removing from transport).', $context + ['error' => $throwable]);
+                    $this->logger->critical('Error thrown while handling message {class}. Removing from transport after {retryCount} retries. Error: "{error}"', $context + ['retryCount' => $retryCount, 'error' => $throwable->getMessage(), 'exception' => $throwable]);
```

Comment:
including the exception message makes sure you can see it without having to run in `-vvv` mode. the context otherwise only shows the exception class but not the message.